### DISCUSSION TITLE
Should the tool use git commit range syntax or flags?

### DISCUSSION THREAD

**Discussion by deweysasser** [deweysasser/changetool#8] [Ideas]
`changetool semver --since-tag v1.0.9`

Or 

`changetool semver v1.0.9..`

Which is the more natural syntax for your use?

**Comment by democracytoday** [comment]
I feel like `--since-tag` explains itself to the noob a bit better, even though `tag..` gets big points for consistency. If you were planning on supporting an `--end-tag`, the git commit range syntax is more natural: `v1.0.9..v2.0.0` is way better than `--since-tag v1.0.9 --end-tag v2.0.0`. However, since it only makes sense to use `--since-tag` with changetool, I say the more verbose flag is preferable. It also doesn't mislead the user into assuming you could do general git commit range syntax things like `v1.0.9..v2.0.0`, since the end tag is what you're generating.
